resources/admin/__init__.py

```diff
@@ -99,22 +99,86 @@ class UnitIdentifierInline(admin.StackedInline):
     fields = ('namespace', 'value')
     extra = 0
 
+class ResourceTagInline(admin.TabularInline):
+    model = ResourceTag
+    fields = ('label',)
+    verbose_name = _('Keyword')
+    verbose_name_plural = _('Keywords')
+    extra = 0
 
 class ResourceAdmin(PopulateCreatedAndModifiedMixin, CommonExcludeMixin, TranslationAdmin, HttpsFriendlyGeoAdmin):
     inlines = [
         PeriodInline,
         ResourceEquipmentInline,
         ResourceGroupInline,
+        ResourceTagInline
     ]
 
     default_lon = 2478871  # Central Railway Station in EPSG:3857
     default_lat = 8501259
     default_zoom = 12
 
-    list_display = ('name', 'unit', 'public', 'reservable')
+    list_display = ('name', 'unit', 'public', 'reservable',)
     list_filter = ('unit', 'public', 'reservable')
     list_select_related = ('unit',)
-    ordering = ('unit', 'name', )
+    ordering = ('unit', 'name',)
+
+    fieldsets = (
+        (None, {
+            'fields': (
+                'unit', 'type',
+                'purposes',
+            ),
+        }),
+        (_('Resource Information'), {
+            'fields': (
+                'public', 'reservable',
+                'name', 'description',
+                'authentication',
+                'min_age', 'max_age',
+                'people_capacity', 'area',
+                'location', 'generic_terms',
+                'payment_terms', 'specific_terms',
+                'access_code_type',
+            ),
+        }),
+        (_('Timmi Information'), {
+            'fields': (
+                'timmi_resource', 'timmi_room_id',
+            ),
+        }),
+        (_('Reservation Information'), {
+            'fields': (
+                'need_manual_confirmation',
+                'reservation_metadata_set',
+                'reservation_home_municipality_set',
+                'cooldown', 'slot_size',
+                'min_period', 'max_period',
+                'max_reservations_per_user',
+                'resource_staff_emails',
+                'reservation_info',
+                'reservation_additional_information',
+                'responsible_contact_info',
+                'reservation_requested_notification_extra',
+                'reservation_confirmed_notification_extra',
+            ),
+        }),
+        (_('Price Information'), {
+            'fields': (
+                'min_price', 'max_price',
+                'price_type', 
+            ),
+        }),
+        (_('Extra Information'), {
+            'fields': (
+                'tags',
+                'external_reservation_url',
+                'id',
+            ),
+        }),
+    )
+
+    readonly_fields = ('tags', )
```

Comment:
Read only?

Reply:
I set the resource model "tags" field to read-only https://github.com/codepointtku/respa/blob/977eb28cdb189d496bbae43a1854d95a458af707/resources/models/resource.py#L230 in favor of using the new resourcetag model https://github.com/codepointtku/respa/blob/977eb28cdb189d496bbae43a1854d95a458af707/resources/models/resource.py#L92 via the inline field https://github.com/codepointtku/respa/blob/6ce84afe11eed3228c77f9dfbceee436d577094b/resources/admin/__init__.py#L102

Reply:
Ok. Good